src/java_bytecode/java_bytecode_convert_class.cpp

```diff
@@ -256,23 +248,21 @@ void java_bytecode_convert_classt::convert(
 /// java::array[X] where X is byte, float, int, char...
 void java_bytecode_convert_classt::add_array_types(symbol_tablet &symbol_table)
 {
-  // this method only adds stuff to the symbol table, no need to execute it more
-  // than once
-  if(add_array_types_executed)
```

Comment:
Is this condition is no longer required as we check symbols are new?

Reply:
Yes, there is now check if symbol is in symbol table.